package.json

```diff
@@ -42,6 +44,21 @@
   },
   "devDependencies": {
     "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
-    "cypress": "^13.6.6"
+    "@storybook/addon-essentials": "^7.6.17",
+    "@storybook/addon-interactions": "^7.6.17",
+    "@storybook/addon-links": "^7.6.17",
+    "@storybook/addon-onboarding": "^1.0.11",
+    "@storybook/blocks": "^7.6.17",
+    "@storybook/preset-create-react-app": "^7.6.17",
+    "@storybook/react": "^7.6.17",
+    "@storybook/react-webpack5": "^7.6.17",
+    "@storybook/test": "^7.6.17",
+    "cypress": "^13.6.6",
+    "eslint-plugin-storybook": "^0.8.0",
+    "normalize.css": "^8.0.1",
+    "prop-types": "^15.8.1",
+    "sass": "^1.71.1",
+    "storybook": "^7.6.17",
+    "webpack": "^5.90.3"
```

Comment:
Do we really need to install webpack here? I think Storybook already has it in the dependencies under the hood. (It's not breaking anything, but it's important to check what dependencies you have in a project.)